Barotrauma/BarotraumaShared/SharedSource/Items/Components/Machines/Reactor.cs

```diff
@@ -376,7 +376,6 @@ static float adjustValueWithoutOverShooting(float current, float target, float s
             item.SendSignal(((int)-CurrPowerConsumption).ToString(), "power_value_out");
             item.SendSignal(((int)Load).ToString(), "load_value_out");
             item.SendSignal(((int)AvailableFuel).ToString(), "fuel_out");
-            item.SendSignal(((int)fuelLeft).ToString(), "fuel_percentage_left");
 
```

Comment:
This should not be removed, because it breaks backwards compatibility with items that still use this connection.

Reply:
Re-added.